grails-app/migrations/install/changelog-create-schema.groovy

```diff
@@ -1600,7 +1604,7 @@ databaseChangeLog = {
 
             column(name: "quantity_ordered", type: "DECIMAL(65)")
 
-            column(name: "adjustments_count", type: "DECIMAL(45)")
+            column(name: "adjustments_count", type: "DECIMAL(44)")
```

Comment:
Can you explain the reason for this change? I'm not saying the original "45" was any better, just don't understand why 44 is any better?

I know these are views (and actually probably need to be removed from the create schema file) but we should actually come up with some sensible defaults for fields. 

* For counts, we probably want this to be an integer. 
* For currency values, I like to use DECIMAL(19, 2). But I just learned that to be compatible with GAAP accounting practices, then you need at least 4 decimal places to the right of the separator. 
* For quantity, eventually I want to use DECIMAL but INT is fine for now, especially for views.

https://stackoverflow.com/questions/13030368/best-data-type-to-store-money-values-in-mysql
https://rietta.com/blog/best-data-types-for-currencymoney-in/

Reply:
As a reminder, the original changelog-create-schema.groovy is not special. We don't need to preserve it. It was created after the first rebase and then recreated for several rebases after that. But we don't need to keep it intact. In fact, I'm starting to think maybe it would be best to break it back up into the different objects (tables, fks, indices, unique constraints, etc)

Reply:
> Can you explain the reason for this change? I'm not saying the original "45" was any better, just don't understand why 44 is any better?

It's what was generated. I don't know why it chose 44 over 45. I can dig that topic a little bit.

Reply:
> changelog-create-schema.groovy is not special. We don't need to preserve it. It was created after the first rebase and then recreated for several rebases after that. But we don't need to keep it intact. In fact, I'm starting to think maybe it would be best to break it back up into the different objects (tables, fks, indices, unique constraints, etc)

Ok, I'll split it into smaller chunks.